test/jdk/java/awt/Mixing/AWT_Mixing/OverlappingTestBase.java

```diff
@@ -355,7 +355,8 @@ protected void clickAndBlink(Robot robot, Point lLoc) {
     protected String failMessage = "The LW component did not received the click.";
 
     private static boolean isValidForPixelCheck(Component component) {
-        if ((component instanceof java.awt.Scrollbar) || isMac && (component instanceof java.awt.Button)) {
+        if ((component instanceof java.awt.Scrollbar)
+                || (isMac && ((component instanceof java.awt.Button) || (component == null)))) {
```

Comment:
"removed pixel check during EmbeddedFrame test from OverlappingTestBase."

Well you presumably can't pixel check a null component, but why is it null ?
It seems like a test issue for it to be null. Can you explain how this happens ?

Also if this code *were* to stay it would be better formatted like
 if ((component instanceof java.awt.Scrollbar) ||
      (isMac && ((component instanceof java.awt.Button) ||
      (component == null))))
 {

Reply:
When component is null it means EmbeddedFrame is currently being tested